### PR TITLE
Update Warframe lines

### PR DESCRIPTION
**Pull request by Nidoskull**
I'd have added Hek's lines from the new Koumei update stuff but they're not on the wiki yet

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Seems to be removing several? Any particular reason?

**Comment by Nidoskull** [comment]
Whoops, I reorganized Ordis's lines to be more easily cross-reference-able with the wiki and missed pasting some back in. Will fix.

File: modules/warframe/data/ordis_strings.txt [hunk 1]
```diff
@@ -1,38 +1,108 @@
...
-**̝͕̺̤̞Y̝̖͕̣͠o̵̟̤͎̣̖u̖ ҉̜a̲͈̭̭̙͕͓r̜͍͈̭̭e̶͔̤ ̱̘̝̝̖t̘̥̦h̢̖̺͔e̳̹͔͟ ̗̟͓̰̀T̟̰̺̥̙͉̮͞en͔̬͓̖̫̪n̵̥o͚̜͈̞̱.͔͖̳̝̳̗ ̗͠ͅY͖͚̗̖̤o͙̟̲u͈̥͉̝ͅ ̭̫͙a̠͈͉͇̥r̛̦̬͔̼͓e̫͞ ̟͍̗ṱ̣̤̗̘h͔͖̤͝e̬͉͟ ҉̥̪̰̗̻̰͔O̦p҉̲͓̯̜̯̳̫e͍͖̟͉̠̦̥r͚̠͢a͈̺̯͍̞̠̮t̛͙̰̤̹̪̼ͅo̸̠r̨̼.̤̗̫̲̫̫͟ ͕̰̳͍͍̖O̶͎͖̠̩r̭͔͙dḭ̹͎̗̞ş̯͎ ̝i҉̯͙͚̟s ̯̜t̺h̜̘̗̘̤̥ȩ͕̮̜͓͉̦ ̡̤̬̖̗̩̤̙C̳̞̥̳e̙p҉̻̙͖̫̟̟ḫ̷̙a̴͕͎l̪̼̜̺̯̱͕͡o̲̙͍n̤̟̩͈̮̤.҉͔ ͓̻̭͕͓̬͍O̺͜ͅrd͍̫i̱̻̰͙̣s̺̫̬͎̥ͅ ̸̙̺̲̟̬͓i̜̳̱s̖̩̀ͅ ̷t̖̭͓̦̩͓h̺̙͍̯͓͕e ͍̩̣̦̣̰͡s̬̤͟h͓̩͇̬ip̹̙.̝̱̖̙̹̙͝ͅ**
+**̝͕̺̤̞Y̝̖͕̣͠o̵̟̤͎̣̖u̖ ҉̜a̲͈̭̭̙͕͓r̜͍͈̭̭e̶͔̤ ̱̘̝̝̖t̘̥̦h̢̖̺͔e̳̹͔͟ ̗̟͓̰̀T̟̰̺̥̙͉̮͞en͔̬͓̖̫̪n̵̥o͚̜͈̞̱.͔͖̳̝̳̗ ̗͠ͅY͖͚̗̖̤o͙̟̲u͈̥͉̝ͅ ̭̫͙a̠͈͉͇̥r̛̦̬͔̼͓e̫͞ ̟͍̗ṱ̣̤̗̘h͔͖̤͝e̬͉͟ ҉̥̪̰̗̻̰͔O̦p҉̲͓̯̜̯̳̫e͍͖̟͉̠̦̥r͚̠͢a͈̺̯͍̞̠̮t̛͙̰̤̹̪̼ͅo̸̠r̨̼.̤̗̫̲̫̫͟ ͕̰̳͍͍̖O̶͎͖̠̩r̭͔͙dḭ̹͎̗̞ş̯͎ ̝i҉̯͙͚̟s ̯̜t̺h̜̘̗̘̤̥ȩ͕̮̜͓͉̦ ̡̤̬̖̗̩̤̙C̳̞̥̳e̙p҉̻̙͖̫̟̟ḫ̷̙a̴͕͎l̪̼̜̺̯̱͕͡o̲̙͍n̤̟̩͈̮̤.҉͔ ͓̻̭͕͓̬͍O̺͜ͅrd͍̫i̱̻̰͙̣s̺̫̬͎̥ͅ ̸̙̺̲̟̬͓i̜̳̱s̖̩̀ͅ ̷t̖̭͓̦̩͓h̺̙͍̯͓͕e ͍̩̣̦̣̰͡s̬̤͟h͓̩͇̬ip̹̙.̝̱̖̙̹̙͝ͅ**Operator, I've been thinking. My misplaced memories and damaged communication systems. What if...Ordis did those things?
```
Comment (MistakeNot4892) [inline modules/warframe/data/ordis_strings.txt]: Accidental missed a newline, looks like.

File: modules/warframe/data/ordis_strings.txt [hunk 2]
```diff
@@ -1,38 +1,108 @@
...
+Do you remember the Old War, Operator? Ordis seems to have... misplaced those memories.
+Operator, the system needs you. Will you begin another mission?
 I've been thinking, Operator...I thought you'd want to know.
-Operator, I hope you are comfortable? No... we do not seem to have any seats.
+Operator! Did you hear that? It said: *kssshhhshhhshhhhhhshhshshht*. Cosmic background radiation is a riot!
+Operator, were you visualizing a bloody battle? **M͏̹͎̙ę͖̺̗ ̭̻̻̰t̺̖̖͚̥͚o͘o̹͈͖̝̳**!
+Stand by while I analyze the intelligence profile of the Grineer. *Error, not a number*! Did the Operator enjoy this witticism?
 Everything in Ordis, Operator? **I̛͕̝̹ͅs͍ ̢͕͍̱̟̼t͡ḩ͉a͎̼̺t̺̯́ a ̼̘̼̺̺̪P̤̦̼͜U͓͍͕͚̳̠͍N̨̪̼̖ͅ?!** Hmm.... I will attempt to bypass this fault.
 Ordis has been counting stars, Operator. All accounted for.
-Operator, I will never betray you. I will keep the Orbiter hidden in the void. You can count on me!
```
Comment (MistakeNot4892) [inline modules/warframe/data/ordis_strings.txt:6]: Missing line.

File: modules/warframe/data/ordis_strings.txt [hunk 3]
```diff
@@ -1,38 +1,108 @@
...
 Operator, did you enjoy that poem? Ordis is composing one as well. It begins: **A̞ ͉̼s̵̬̼c͖̫͙͍̭̲͡ͅu͇̳̖͕̳m͖ͅb͉̙̰͘a̟̝̜͎͚̮͔g̞̣̝̪ ̴̦̘̟͓̹̟̠C̛̤̱̭ọ̲̯̲͘r̶p̲͉͟u̬̤s̲̟ ̧̫ͅf̨̠̰͖̠̪̯r͓͎̟̱̬̪̰̀o̧͙̯̤̱m̻̲̩̯̩ ̩̤̝̟͙̞V̖̤͟ḛ̠̲͍͙nu̧͖̜s̟͎͘ͅͅ...**
 The Operator comes first! **S̮͕̠̱͙H̥̫̻͎̭͖́ͅU̴̙̟̱͈͓̳ͅŢ̹͔̦̟ Y͚͍͖̹̞O̶̪̲̹̪̻U͕̗̭̫̺̟R̵̤͔ ͔̣̱̻͔OS̡͓̹̭̻͇ͅC͖͇̬I͕̙͓̹̟̞L͈L͡A̗͓͍T̮̪̭͓̲̰͡ÓR̝͙ ͈S̶I̮̣̩͉̥̤M͈͔ÁR̲̞͔̗͍̥̜I̠͜S͓͍̥͜!**
-My Operator is no stranger to **s̟̹h̸͙͈̩̭̜r̢̖̗̳ȩ̘̖ͅd͏̬̞̳d͎̠i͎̹̤̞͓̹͞n̯̪̜̠͕g͍͓̗̙̼ ̵̩f̞̼̮͙̠l͙͕̣̩̟ḛ͖̰̤̗̞sh̜** eradicating infestations.
 Ordis remembers many more Cephalon voices, but now it is only silence...
 Tenno, **R҉ÌP̗͓͙̞ ͙̟̯IT̳ ̧̗̺̹͕̩A̝̜̲̯P̶͈A̭̝̦͙̹ͅR̴̜̫͓͖̤͎̠T̠̹̬͠**.
-**N̤̰̹̭O̬̤̰̬͈͇̦͟B̫͍̫͠O̭D̖͢Ỵ̤͇̱ ̮̻͇̲̥͞M̖̺̻͕͢A̧̤̩͙K̖̱̟͓͚̯͘E͔S ̙͔̦̘̣̪͕̕Ạ̦ ͕F̜͈̘̼OO̩͖̭̰̩̩̰Ĺ̟͎̭̱̦͇ Ơ̪͕̟̻̫F̵̜̟͎̜̹ ͚O͈̞̗͎̥ͅR̭͞DI̻͍͞S̩̬̦͈̬͇͝!** Oh dear, where did that come from?
-Bypass that console to overdrive the system. Try to increase the amplification circuits **T̰̣͈͕̦O̲͇͟.̬ ͡E̸̟̳̖̠̲L̙͖̲̻͔EV̟̣̯̟E͏̬͎̙̝N̺̮̜̜̞.**
```
Comment (MistakeNot4892) [inline modules/warframe/data/ordis_strings.txt:37]: Missing line.

**Comment by MistakeNot4892** [comment]
Couple of missing lines and one formatting error, will merge after those are addressed.

File: modules/warframe/data/ordis_strings.txt (hunk 3 above)
Comment (Nidoskull) [reply modules/warframe/data/ordis_strings.txt:37]: Line 36 is re-zalgo'd and therefore GitHub thinks it's missing entirely. It's on line 49 now.
The other one was indeed missing, thanks.

**Pull request merged**